src/test/resources/ant/Resolve/pom.xml

```diff
@@ -37,6 +37,16 @@
     <aetherVersion>0.9.0.M3</aetherVersion>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.apache.maven.resolver</groupId>
+        <artifactId>maven-resolver-api</artifactId>
+        <version>1.8.0</version>
```

Comment:
This looks wromg, resolver api 1.8.0, but util 1.7.0.... I know this is test, but could this be updated?

Reply:
@cstamas, what pair of dependencies do you suggest?